open-sphere-plugins/feature-actions/src/main/java/io/opensphere/featureactions/editor/model/FeatureActionImporter.java

```diff
@@ -0,0 +1,213 @@
+    {
+        return null;
+    }
+
+    @Override
+    public String getName()
+    {
+        return "Feature Actions";
+    }
+
+    @Override
+    public int getPrecedence()
+    {
+        return 875;
+    }
+
+    @Override
+    public List<String> getSupportedFileExtensions()
+    {
+        return SUPPORTED_EXT;
+    }
+
+    @Override
+    public void importFile(File file, ImportCallback callback)
+    {
+        boolean success = false;
+        try
+        {
+            NodeList nodeList = getFeatureActionNodes(file);
+            if (nodeList == null)
+            {
+                callback.fileImportComplete(success, file, null);
+                EventQueueUtilities.invokeLater(() -> JOptionPane.showMessageDialog(myParent, "Selected file failed to import."));
+                return;
+            }
+            for (int i = 0; i < nodeList.getLength(); i++)
+            {
+                FeatureAction featureAction = XMLUtilities.readXMLObject(nodeList.item(i), FeatureAction.class);
+                for (SimpleFeatureActionGroup simpleGroup : myModel.getFeatureGroups())
+                {
+                    if (simpleGroup.getGroupName().equals(featureAction.getGroupName()))
+                    {
+                        Platform.runLater(() -> simpleGroup.getActions().add(new SimpleFeatureAction(featureAction)));
+                    }
+                }
+            }
+            success = true;
+        }
+        catch (JAXBException e)
+        {
+            LOGGER.error(e.getMessage(), e);
+        }
+        if (callback != null)
+        {
+            callback.fileImportComplete(success, file, null);
+        }
+    }
+
+    @Override
+    public void importFiles(List<File> fileList, ImportCallback callback)
+    {
+    }
+
+    @Override
+    public boolean importsFileGroups()
+    {
+        return false;
+    }
+
+    @Override
+    public boolean importsFiles()
+    {
+        return true;
+    }
+
+    @Override
+    public boolean importsURLs()
+    {
+        return false;
+    }
+
+    @Override
+    public void importURL(URL url, Component component)
+    {
+    }
+
+    @Override
+    public void importURL(URL url, ImportCallback callback)
+    {
+    }
+
+    /**
+     * Gets a list of feature action nodes from the file.
+     * @param file The file to get feature actions from.
+     * @return The list of nodes of feature actions.
+     */
+    private NodeList getFeatureActionNodes(File file)
+    {
+        NodeList nodes;
+        try
+        {
+            DocumentBuilder docBuilder = XMLUtilities.newDocumentBuilderNS();
+            Document doc = docBuilder.parse(file);
+            nodes =  StateXML.getChildNodes(doc, "/:featureActions/featureAction");
+        }
+        catch (ParserConfigurationException | SAXException | IOException | XPathExpressionException e)
+        {
+            nodes = null;
+            LOGGER.error(e.getMessage(), e);
```

Comment:
instead of logging the e.getMessage, log what produced it. like: 
`"Unable to parse feature action"`